src/net_processing.h

```diff
@@ -72,6 +72,11 @@ struct Peer {
 
     /** This peer's reported block height when we connected */
     std::atomic<int> m_starting_height{-1};
+    /** The final block hash in an INV message. When the peer requests this
+     * block, we send a BLOCK message to trigger the peer to request the next
```

Comment:
We send an INV message, not a BLOCK message.

Reply:
Yes, will fix.

Reply:
Done.